distribution/kernel/carbon-home/repository/resources/conf/default.json

```diff
@@ -153,5 +153,22 @@
   "truststore.password": "wso2carbon",
   "encryption.type": "asymmetric",
   "encryption.algorithm": "AES",
-  "encryption.alias": "symmetric.key.value"
-}
\ No newline at end of file
+  "encryption.alias": "symmetric.key.value",
+
+  "authenticator.token_ui_authenticator.disable": true,
+  "authenticator.token_ui_authenticator.priority": "5",
+
+  "authenticator.saml_sso_authenticator.disable": true,
```

Comment:
How about a name like,
"admin_console.authenticator.saml.enable":false ?

"authenticator" itself can be misread for authentication framework authenticators